whctools/models.py

```diff
@@ -4,8 +4,8 @@
 from django import forms
 
 from allianceauth.eveonline.models import EveCharacter
-
-
+from allianceauth.framework.api.evecharacter import get_main_character_from_evecharacter
```

Comment:
I believe "framework" is an AA 4.0 only thing. So this will fail right now until we upgrade.
If you look at my original code you can see that I use the "parent" to get the main char for this reason. 
So either we wait until we upgrade to 4.0 or you could put in a work around for now. Ill leave that up to you.